README.md

```diff
@@ -20,15 +20,15 @@ Once you've run the above commands the pre-commit framework will be installed wi
 
 ## Usage
 
-The first time you run `git commit` **in a repository** will throw a warning to tell you that you need to create a secrets baseline, as shown below:
+The first time you run `git commit` in a repository will throw a warning to tell you that you need to create a secrets baseline, as shown below:
 
 ```shell
 Unable to open baseline file: REPO_ROOT/.secrets.baseline
 Please create it via
-   detect-secrets scan > $HOME/<your-git-repo>/.secrets.baseline
+   detect-secrets scan > /<path-to-your-git-repo>/.secrets.baseline
 ```
 
-Once you've added your secrets baseline, the first time you run `git commit` **on your machine**, will install the hooks that have been added to your global config.
+Once you've added your secrets baseline, the first time you run `git commit` on your machine, will install the hooks that have been added to your global config.
```

Comment:
Same thing as before - I think it needs to say "**it** will install the hooks..."